client/src/ui/molecules/PeoplePickerSelector.test.tsx

```diff
@@ -128,7 +128,9 @@ describe('PeoplePickerSelector', (): void => {
                 isShowing={true}
             />,
         );
-        expect(baseElement.querySelector('.people-picker__guidance').textContent).toBe('validation--peoplepicker_guidance-prefix ' + min + ' validation--peoplepicker_guidance-suffix');
+        expect(baseElement.querySelector('.people-picker__guidance').textContent).toBe(
+            'validation--peoplepicker_guidance-prefix ' + min + ' validation--peoplepicker_guidance-suffix',
```

Comment:
Are we not localizing the numeric system too?

Reply:
Our internationalisation library doesn't support parameters passed to translations yet so we're going to have to have this type of thing as a limitation for now 👍

Reply:
@Cooryd
Shouldn't this do the trick?
https://www.i18next.com/translation-function/interpolation